src/krux/firmware.py

```diff
@@ -204,21 +204,6 @@ def status_text(text, highlight_prefix=""):
     firmware_hash = sha256(firmware_path)
     firmware_with_header_hash = sha256(firmware_path, new_size)
 
-    status_text(
-        t("New firmware detected.")
-        + "\n\n"
-        + "SHA256:\n\n"
-        + binascii.hexlify(firmware_hash).decode()
-        + "\n\n\n"
-        + t("Install?"),
-        ":",
-    )
-    inp.buttons_active = True
-    if inp.wait_for_button() in (BUTTON_PAGE, BUTTON_PAGE_PREV):
-        display.clear()
-        inp.wait_for_release()  # Wait for button release loading inputs on context
-        return False
-
```

Comment:
The only issue I see removing this, is the device "frozen" with a black screen for a longer time. To fix this we could just display "Processing.." here.

Reply:
Where exactly does a frozen black screen occur?

Reply:
My bad! There is already a msg on line 201 for that!

> status_text(t("New firmware detected.") + "\n\n" + t("Verifying.."))